src/arrayMethodSort.js

```diff
@@ -4,8 +4,24 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(callback) {
```

Comment:
move default compareFunction to a variable
```suggestion
  [].__proto__.sort2 = function(callback = compareByDefault) {
```